packages/react-app/src/components/icons/QuestionIcon.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint react/jsx-props-no-spreading: off */
+// ☝️ we want this component to be usable with chakra props
+import React from "react";
+import { chakra, useColorMode } from "@chakra-ui/react";
+
+const QuestionIcon = props => {
+  const { colorMode } = useColorMode();
+
+  return (
+    <chakra.svg width="44" height="44" {...props} viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M43.3266 4.63367H41.2701V39.6749H43.3266V4.63367Z"
+        fill={colorMode === "light" ? "#026262" : "#C8F5FF"}
```

Comment:
Currently rendering hardcoded values in `questionIcon` based on dark or light mode. 
**Reason** : 
Using hardcoded hex string because it was not recognizing "sre.default" and "sreDark.default", Event tried using `chakra.path` but there is some ambiguity between `d` attribute of `path` and chakra inferring `d` as `display` more on this [here](https://github.com/chakra-ui/chakra-ui/issues/2620) 

**Possible Solution** : 
maybe creating a hook similar to `useCustomColorModes` but instead, it returns the hex string (eg. `"#026262"`) of that color instead of `"sre.default"` 

generalising it because we may use it for other svg icons too....please let me know what you think or any other approach we should take for this

Reply:
It seems that you solved this, right? I see you are using it like `var(--chakra-colors-sre-text)`. I think another approach that might be a bit cleaner is using `useColorModeValue`

e.g on `QuestionIcon`:


```js
const iconFill = useColorModeValue("sre.text", "sreDark.text");

# And then using the iconFill on the fill prop
# instead of fill={colorMode === "light" ? "var(--chakra-colors-sre-text)" : "var(--chakra-colors-sreDark-text)"}
# you get fill={iconFill}
```

Also sharing some thoughts here:

- When you have SVGs as a component it's great to be able to modify props (as we need for dark/light themes). The "downside" can be that we are increasing the JS bundle size, since this SVG are embedded in the code.(which load fast!)

vs.

- Having the SVG as a "public" file that you can request for your front-end, it saves to bundle size (but you have to request a file!). Also it's trickier when you want to have dynamic props: you could have a "class" on the SVG and having CSS with the fill for each class depending on the color mode.

In this case, let's not overthink it, since there are only a few. Just wanted to put this on the table :P

Reply:
Tysm for the insights !! really helpful and noted it 🙏  !!  

Regarding the `QuestionIcon ` I tried using `useColorModeValue ` : 

```js
const iconFill = useColorModeValue("sre.text", "sreDark.text");
console.log("⚡️ ~ file: QuestionIcon.jsx ~ line 9 ~ QuestionIcon ~ iconFill", iconFill);
```
output : 
```js
⚡️ ~ file: QuestionIcon.jsx ~ line 9 ~ QuestionIcon ~ iconFill sreDark.text
```

But I don't know why chakra seems to give me `"sreDark.text"` as return value of `useColorModeValue` isntead of `hex  color string`

Reply:
> But I don't know why chakra seems to give me "sreDark.text" as return value of useColorModeValue isntead of hex color string

This is the expected behavior. But when you pass that to a Chakra component... it works! (like `<chakra.path`)

But, now I see this: 

> Event tried using chakra.path but there is some ambiguity between d attribute of path and chakra inferring d as display more on this https://github.com/chakra-ui/chakra-ui/issues/2620

So.. yeah, let's forget about this.

Thanks!